src/source/PeerConnection/PeerConnection.c

```diff
@@ -844,6 +844,12 @@ STATUS setRemoteDescription(PRtcPeerConnection pPeerConnection, PRtcSessionDescr
     CHK_STATUS(setTransceiverPayloadTypes(pKvsPeerConnection->pCodecTable, pKvsPeerConnection->pRtxTable, pKvsPeerConnection->pTransceievers));
     CHK_STATUS(setReceiversSsrc(pSessionDescription, pKvsPeerConnection->pTransceievers));
 
+    if (getenv(DEBUG_LOG_SDP)) {
+        DLOGD("******************REMOTE SDP********************");
+        DLOGD("%s\n", pSessionDescriptionInit->sdp);
+        DLOGD("************************************************");
```

Comment:
Is it possible that we will have multi-thread case causing the log lines to be interacting each other? For example:
```
******************REMOTE SDP********************
******************LOCAL SDP********************
(local SDP)
(remote SDP)
************************************************
************************************************
```

Reply:
Potentially. I changed it to print LOCAL_SDP,...etc along with the sdp printing line.